autosklearn/data/target_validator.py

```diff
@@ -386,3 +386,10 @@ def _check_data(
                                  self.type_of_target,
                                  supported_output_types
                              ))
+
+    @property
+    def classes(self) -> typing.List:
```

Comment:
Shouldn't this be `classes_`?

Reply:
This line is the one that needs to be classes https://github.com/automl/auto-sklearn/blob/738f66ccf8f672d56b0c43ca62e9c4d707d2bd48/autosklearn/estimators.py#L604

I can make this function classes_ if you think it makes sense to have the same attribute everywhere.

Reply:
Well, the idea of `classes_` is to store the classes after fit, adhering to scikit-learn's best practices by having a trailing `_`. As this function does exactly this, yes, I think so.